src/llmcompressor/evaluation/kld.py

```
@@ -0,0 +1,501 @@
+            raise ValueError(
+                f"Mismatched prompt counts: base={len(base_hidden)}, "
+                f"quant={len(quant_hidden)}"
+            )
+
+        lm_head.eval()
+        per_prompt_kld: list[float] = []
+        total_tokens = 0
+        skipped = 0
+
+        with torch.no_grad():
+            for idx, (h_base, h_quant) in enumerate(zip(base_hidden, quant_hidden)):
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)